src/index.ts

```diff
@@ -2,7 +2,7 @@ import { getPayouts, PayoutDetails, PayoutTransaction } from "./core/payouts";
 import { getStakes } from "./core/stakes";
 import { getBlocks, getLatestHeight } from "./core/queries";
 
-import CodaSDK, { keypair } from "@o1labs/client-sdk";
+import { keypair } from "mainnet-client-sdk";
```

Comment:
Assume keypair is stable across - find to just use from mainnet sdk? Seems okay.

Reply:
I think the `keypair`and `payment` type are not changed so I reused it from the mainnet SDK client.
Do you have any method to choose the `type` conditionally?